server/helpers/utils.ts

```diff
@@ -105,3 +107,16 @@ export function formatMessage(message) {
     }
   ];
 }
+
+export async function isSpaceAdmin(spaceName, address) {
+  if (
+    spaces[spaceName].admins &&
+    spaces[spaceName].admins
+      .map(admin => admin.toLowerCase())
+      .includes(address.toLowerCase())
+  ) {
+    return true;
+  }
+  const spaceUri = await getSpaceUri(spaceName);
```

Comment:
We should include the address from the IPNS url here, only the admins.

Reply:
You dont need a helper function you can call this directly on the verify function